### PR TITLE
Add section on Interaction Types.

### PR DESCRIPTION
**Pull request by msporny**
This PR addresses a request raised by @OR13 in https://github.com/w3c-ccg/vc-api/pull/254#issuecomment-1008442749 and adds details about the interact property in a Verifiable Presentation Request.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vp-request-spec/pull/13.html" title="Last updated on Jan 15, 2022, 4:03 PM UTC (77d7915)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c-ccg/vp-request-spec/13/3c56d93...77d7915.html" title="Last updated on Jan 15, 2022, 4:03 PM UTC (77d7915)">Diff</a>

### CODE REVIEW

**Review by OR13 — APPROVED** [review]

**Review by dmitrizagidulin — APPROVED** [review]
:+1:

File: index.html [hunk 1]
```diff
@@ -314,11 +321,60 @@ <h3>Authorization Capability Request</h3>
...
+The <a>interaction type</a> serves as the main extension point mechanism for
+ways of responding to a query.
+      </p>
+
+      <section>
+        <h3>Mediated Presentation</h3>
+
+        <p>
+A mediated presentation service requires an interaction by an entity,
+typically a person, that can interact with an interface.
```
Comment (dlongley) [inline index.html:339]:
```suggestion
A mediated presentation service requires the use of an out-of-band
interface, for example, a person using a Web browser.
```

File: index.html [hunk 2]
```diff
@@ -314,11 +321,60 @@ <h3>Authorization Capability Request</h3>
...
+...
+        </pre>
+      </section>
+
+      <section>
+        <h3>Unmediated Presentation</h3>
+
+        <p>
+An unmediated presentation service can process a presentation in an entirely
+automated fashion.
```
Comment (dlongley) [inline index.html:359]:
```suggestion
An unmediated presentation service requires no out-of-band interfaces,
enabling fully-automated presentation processing.
```

**Review by dlongley — APPROVED** [review]
Approving w/suggestions.

**Pull request merged**